ci/pipelines/deploy-to-test.yml

```diff
@@ -1,3 +1,40 @@
+definitions:
```

Comment:
These are anchors because we're going to reuse the same large unchanging blobs in every copy